ui/job-view/JobView.jsx

```diff
@@ -1,8 +1,6 @@
 import React from 'react';
-import { react2angular } from 'react2angular/index.es2015';
 import SplitPane from 'react-split-pane';
 
-import treeherder from '../js/treeherder';
```

Comment:
In case you were wondering why this removal didn't result in a big size decrease from the built asset sizes ([before](https://travis-ci.org/mozilla/treeherder/jobs/445096676#L702-L703) and [after](https://travis-ci.org/mozilla/treeherder/jobs/444809745#L713-L714)), it's because angular is still listed in the vendor list in `neutrino-custom/base.js`, which forces it to be in the shared vendor chunk.

With Neutrino 9 we no longer need to maintain that list, so we'll see the size benefits regardless -- however if you wanted in the meantime to remove the 4 angular-related packages from the vendor list, the bundle size would decrease.